data.go

```diff
@@ -53,6 +54,27 @@ func ToIngressifyRule(il *v1beta1.IngressList) []IngressifyRule {
 	return ifyrules
 }
 
+// IngRules is just an alias to be able to implement custom sorting.
+type IngRules []IngressifyRule
+
+func (ir IngRules) Len() int {
+	return len(ir)
+}
+
+func (ir IngRules) Swap(i, j int) {
+	ir[i], ir[j] = ir[j], ir[i]
+}
+
+func (ir IngRules) Less(i, j int) bool {
+	return (len(ir[i].Path) > len(ir[j].Path))
+}
+
+// OrderAscByPathLen order the rules by Path length in ascending order
+func OrderAscByPathLen(rules []IngressifyRule) []IngressifyRule {
```

Comment:
Any function that goes into the template is fundamentally a *contract* :) Because we can't afford to change any argument or parameter in it, as we have to preserve compatibility, even during development :)

So if exposing it as a template function - Would you like to have a nice name from beginning ;)

`OrderByPath(rules, asc=true/false)` or `OrderByPath(rules, options)` or something like that ;)

Reply:
sounds like a good idea. I propose we stick to the name `OrderByPathLen` just to be clear that this isn't lexicographical order and we convey order `asc/desc` through a function param.